### PR TITLE
Fix tutorial nb

### PR DESCRIPTION
**Pull request by daviddoji**
This change fixes the tutorial 1 notebook.

@steinnymir please have a look and merge if you agree with the changes

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -12,31 +12,30 @@ repos:
...
         args: [--py36-plus]
-- repo: https://gitlab.com/iamlikeme/nbhooks
-  rev: 1.0.0
-  hooks:
-  - id: nb-ensure-clean
-    args: [--meta, ExecuteTime]
+-   repo: https://github.com/kynan/nbstripout
+    rev: 0.6.0
+    hooks:
+    -   id: nbstripout
```
Comment (rettigl) [inline .pre-commit-config.yaml:41]: Does this automatically clean the output? Do we want this? Do we not also want to allow to publish notebooks with the expected output present?

Comment (rettigl) [reply .pre-commit-config.yaml:41]: IMHO the information of the old precommit hook that the notebook still contains output is sufficient. One can then decide to still keep it, or manually strip it. Just my 5 cent.

Comment (daviddoji) [reply .pre-commit-config.yaml:41]: yes, it does. This is typical way of sharing a notebook, as not all renders work in the same way.

**Comment by daviddoji** [comment]
There is a test failing looking for a function in the `diagnostics` module that is not there.

**Comment by rettigl** [comment]
This is fixed in the other branch. Merge the other PR first, then merge main into this one. Then it should be fine.

**Comment by daviddoji** [comment]
> This is fixed in the other branch. Merge the other PR first, then merge main into this one. Then it should be fine.

Could you guys please merge the branch with the fix into main? Main should be bug free!

**Comment by steinnymir** [comment]
@daviddoji please fix the conflict and I'll merge it

**Comment by daviddoji** [comment]
Usually, fixes are done only by the person who opens the PR. Otherwise, you duplicate the work ;)

**Pull request merged**